server/src/config/mongoose.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable global-require */
+const mongoose = require('mongoose');
+
+const config = require('./index').mongo;
```

Comment:
nitpick: I noticed this when working on this branch, but I found it confusing that this is variable is called config when it's actually just the mongo config - can we either consistently call this something other than config, or to just do something like:

const config = require('./index');
and then use config.mongo.uri?

Reply:
Fixed & pushed for those files:
- src\config\mongoose.js
- src\services\queue.js
- src\services\crawler\twitter\index.js